Rakefile

```diff
@@ -14,66 +14,48 @@ task :generate_csv do
     "ocd-division/country:ug/" + parts.map { |type, value| [type, idify(value)].join(':') }.join('/')
   end
 
-  csv_url = 'https://docs.google.com/spreadsheets/d/1LetNFNq6ovg4bbq-Whze0Q06CYSpNqpRIzSUb9yVtw4/export?format=csv'
-
-  csv = CSV.parse(open(csv_url).read, headers: true, header_converters: :symbol)
-
-  ocd_ids = Set.new
-
-  csv.each do |r|
-    next unless r[:arearegion]
-    ocd_ids << {
-      id: id_for(region: r[:arearegion]),
-      name: r[:arearegion]
-    }
-
-    next unless r[:areasub_region]
-    ocd_ids << {
-      id: id_for(region: r[:arearegion], subregion: r[:areasub_region]),
-      name: r[:areasub_region]
-    }
-
-    next unless r[:areadistrict]
-    ocd_ids << {
-      id: id_for(region: r[:arearegion], subregion: r[:areasub_region], district: r[:areadistrict]),
-      name: r[:areadistrict]
-    }
-
-    next unless r[:areaconstituency]
-    ocd_ids << {
-      id: id_for(region:r[:arearegion], subregion: r[:areasub_region], district: r[:areadistrict], constituency: r[:areaconstituency]),
-      name: r[:areaconstituency]
-    }
+  def ocd_ids_from_csv(csv_data, mapping = {})
```

Comment:
Perhaps better to call this `raw_csv_data` or something to make it obvious that the function is going to parse it itself?</nit>

Some documentation would probably be helpful here too, particularly to explain what way around the mapping works (i.e. whether it's "column_name_in_sheet => column_name_we_wanted" or "column_name_we_want => where_to_find_it_in_sheet")